addons/mail/models/mail_thread.py

```diff
@@ -601,6 +606,8 @@ def _message_track_post_template(self, changes):
         for _field_name, (template, post_kwargs) in templates.items():
             if not template:
                 continue
+            # make discussion so followers see responses (not message_type comment to prevent editing)
```

Comment:
shouldn't we have the right values from `_track_template` ? (here you will break all overwrite of `_track_template`, like this one)
```
'subtype_id': self.env['ir.model.data']._xmlid_to_res_id('appointment.mt_calendar_event_canceled')
```